server/src/main/java/com/scalar/db/server/ScalarDbServer.java

```diff
@@ -107,6 +107,9 @@ public void start() throws IOException {
               + "\" to 'jdbc'");
     }
 
+    config.getGrpcMaxInboundMessageSize().ifPresent(builder::maxInboundMessageSize);
+    config.getGrpcMaxInboundMetadataSize().ifPresent(builder::maxInboundMetadataSize);
```

Comment:
We set the configurations to the server here.

Reply:
It means GrpcConfig and GrpcUtils aren't used in `server`, right?

Reply:
Yes, `GrpcConfig` and `GrpcUtils` are for the client side, and they are not used on the `server` side. The class for the configurations for the `server` side is `ServerConfig` actually.